diff --git a/node_modules/@sentry/utils/cjs/instrument.js b/node_modules/@sentry/utils/cjs/instrument.js
index bf15994..792724b 100644
--- a/node_modules/@sentry/utils/cjs/instrument.js
+++ b/node_modules/@sentry/utils/cjs/instrument.js
@@ -492,7 +492,7 @@ function makeDOMEventHandler(handler, globalListener = false) {
 
 /** JSDoc */
 function instrumentDOM() {
-  if (!('document' in WINDOW)) {
+  if (!(WINDOW.document)) {
     return;
   }
 
diff --git a/node_modules/@sentry/utils/esm/instrument.js b/node_modules/@sentry/utils/esm/instrument.js
index a560d7a..648dfe0 100644
--- a/node_modules/@sentry/utils/esm/instrument.js
+++ b/node_modules/@sentry/utils/esm/instrument.js
@@ -490,7 +490,7 @@ function makeDOMEventHandler(handler, globalListener = false) {
 
 /** JSDoc */
 function instrumentDOM() {
-  if (!('document' in WINDOW)) {
+  if (!(WINDOW.document)) {
     return;
   }
 
